EG-1225: Add a link to cordapp-overview, fix some formatting (#46)
corda/corda-docs: content/en/docs/corda-os/4.4/release-notes.md

# Release notes

Welcome to the Corda 4.4 release notes. Please read these carefully to understand what’s new in this release and how the
features can help you. Just as prior releases have brought with them commitments to wire and API stability, Corda 4.4
comes with those same guarantees. States and apps valid in Corda 3.0 are usable in Corda 4.4.

## Corda 4.4

> Corda 4.4 lays the foundation of a new open-core approach for the Corda codebase. This involved a refactoring of the

Reviewer: In line with style guide discussions for open source (as opposed to open-source), I feel we should make open-core open core too?

> Corda 4.4 lays the foundation of a new open-core approach for the Corda codebase. This involved a refactoring of the

Author: ```suggestion
Corda 4.4 lays the foundation of a new open core approach for the Corda codebase. This involved a refactoring of the
```

main functional components of Corda. Please consult [the CorDapp overview](cordapp-overview.md/) to get an overview of
the practical impact on CorDapp development.

> Furthermore, Corda 4.4 introduces improvements to the flow framework API, a new diagnostic `ServiceHub` call and includes

Reviewer: Shouldn't that be
Furthermore, Corda 4.4 introduces improvements to the flow framework API and a new diagnostic `ServiceHub` call, and includes...

Author: ```suggestion
Furthermore, Corda 4.4 introduces improvements to the flow framework API and a new diagnostic `ServiceHub` call, and includes
```

a number of security enhancements.

### Changes for developers in Corda 4.4

#### Flows API improvements

Corda 4.4 introduces a `FlowLogic.await` API that allows a CorDapp developer to suspend their flow when executing
user-defined long-running operations such as call-outs to external services. The `FlowLogic.await` API prevents these

> long-running operations from blocking the flow thread, allowing other flows to progress. Previously, these operations

Reviewer: a few missing commas here in my view (a legacy issue surely - "engineers write docs")

> Corda 4.4 introduces a `FlowLogic.await` API that allows a CorDapp developer to suspend their flow when executing
> user-defined long-running operations such as call-outs to external services. The `FlowLogic.await` API prevents these
> long-running operations from blocking the flow thread, allowing other flows to progress. Previously, these operations

Author: ```suggestion
Corda 4.4 introduces a `FlowLogic.await` API, allowing a CorDapp developer to suspend a flow when executing
user-defined long-running operations, such as call-outs to external services. The `FlowLogic.await` API prevents these
long-running operations from blocking the flow thread, allowing other flows to progress. Previously, these operations
```

had to be executed synchronously, blocking the flow thread.

The CorDapp developer can decide whether to run these asynchronous flow operations in a dedicated thread pool, or to
handle the threading themselves directly.

> Note that as before, the flow framework suspends automatically for certain operations (e.g. when waiting to receive a

Reviewer: e.g. > for example, ...

Author: ```suggestion
Note that as before, the flow framework suspends automatically for certain operations (for example, when waiting to receive a
```
